captum/insights/frontend/src/App.tsx

```diff
@@ -45,7 +53,16 @@ function Visualizations(props) {
   );
 }
 
-class AppBase extends React.Component {
+interface AppBaseProps {
+  fetchInit: () => void;
+  fetchData: (filter_config: FilterConfig) => void;
+  config: any;
```

Comment:
should this be of type InsightsConfig?

Reply:
`InsightsConfig` whole application config. `FilterConfig` is just for the filter part.